product_links/product_links.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Guewen Baconnier
+#    Copyright 2011-2013 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields, api
+
+
+class ProductLink(models.Model):
+    _name = 'product.link'
+    _rec_name = 'linked_product_id'
+
+    @api.model
+    def get_link_type_selection(self):
+        # selection can be inherited and extended
+        return [('cross_sell', 'Cross-Sell'),
+                ('up_sell', 'Up-Sell'),
+                ('related', 'Related')]
+
+    @api.model
+    def _get_link_type_selection(self):
+        return self.get_link_type_selection()
```

Comment:
This one is useless in v8 (see https://github.com/OCA/e-commerce/pull/40#discussion_r31216690)